robotCompletionFinal.cpp

```diff
@@ -0,0 +1,143 @@
+#include "ImageProcessor.hpp"
+
+int core(){
```

Comment:
Since these functions don't return anything, shouldn't their return type be set to void?